### PR TITLE
[Urgent] Improved package install script and manual for new CI server

### PR DESCRIPTION
**Pull request by leemgs**
Fixed issue nnsuite/nnstreamer#887.

This commit is to improve the existing script to install required packages
that are installed in a Ubuntu-based CI server.

**Changes proposed in this PR:**
1. Added graphviz statement as a mandatory requirement for new Ubuntu PC.
2. Update the existing script to install required packages.
3. Removed out-of-date sections in the installation manual.
4. Renamed two scripts for readability and maintenance.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

File: ci/taos/webapp/install-packages-base.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        exit 1
+    else
+        echo -e "[DEBUG] It's okay. All packages are successfully installed...."
+        echo -e "[DEBUG] Then, install the TAOS-CI software as a next step."
+    fi
+}
+
+echo -e "\n\n\n##########  for CI-server: Installing Apache and PHP packages for webhook handler"
+sudo apt -y install bash php curl
+func-pack-check
```
Comment (myungjoo) [inline ci/taos/webapp/install-packages-base.sh]: Usually,
```
do_something || fail
```
is better than
```
do_something
if [[ $? -ne 0 ]]; then fail; fi
```

Comment (myungjoo) [reply ci/taos/webapp/install-packages-base.sh]: Or do single-line apt install:
```
sudo apt -y install ${a_list_of_everything}
...
```

Comment (leemgs) [reply ci/taos/webapp/install-packages-base.sh]: Good comment. Improved. Resubmitted.

**Comment by leemgs** [comment]
**Changes proposed in this PR:**
* Version 2:
1. Simplified a check routine of installation result.

* Version 1:
1. Added graphviz statement as a mandatory requirement for new Ubuntu PC.
2. Update the existing script to install required packages.
3. Removed out-of-date sections in the installation manual.
4. Renamed two scripts for readability and maintenance.

**Comment by leemgs** [comment]
The local test and aging test is successfully completed. This PR is ready to be merged.

**Pull request merged**